sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb

```diff
@@ -283,15 +286,17 @@
    "outputs": [],
    "source": [
     "import sagemaker\n",
+    "\n",
     "sagemaker.Session().delete_endpoint(kmeans_predictor.endpoint)"
    ]
   }
  ],
  "metadata": {
+  "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "conda_python3",
+   "display_name": "Python 3 (Data Science)",
    "language": "python",
-   "name": "conda_python3"
+   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-west-2:236514542706:image/datascience-1.0"
```

Comment:
With the kernel changing to Studio's, you will want to make sure that the intro says which kernel to use... because it'll prompt folks now. An for 1PA, it's not entirely clear what kernel you should use.

Reply:
We're working on a solution to automate kernel selection in JumpStart so that the pop-up won't appear.

Reply:
Made a comment about this in all notebooks.